src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -454,10 +451,10 @@ void runtime::cpu::CPU_ExternalFunction::compile()
     pass_manager.register_pass<ngraph::pass::PropagateCacheability>(
         runtime::cpu::get_annotations_factory());
     pass_manager.register_pass<runtime::cpu::pass::CPUMemoryAssignment>(
-        key_to_tensors_set_map,
-        tensor_to_key_map,
+        bufferID_to_tensorSets,
+        tensor_to_bufferID,
         size_t(s_memory_pool_alignment),
-        !m_pass_config.get_reuse_memory());
+        pass_config.get_reuse_memory());
```

Comment:
Just to make sure that removing the '!' is not unintentional :). Same in line 1168.

Reply:
thanks, you caught it!